manager/controllers/app/blueprint_controller.go

```diff
@@ -297,6 +308,8 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, log logr.Logger, bl
 		}
 		module.Arguments.Labels[app.BlueprintNameLabel] = blueprint.Name
 		module.Arguments.Labels[app.BlueprintNamespaceLabel] = blueprint.Namespace
+		module.Arguments.Labels[utils.FYBRIKAPPUUID] = uuid // used for log correlation
```

Comment:
Wouldn't it be better to define all label constants in one place? Also, I believe the convention is not to use all capital letters, I got a warning about it once.

Reply:
FYBRIKAPPUID will be used in both the control plane and in the data plane, so I didn't think it belonged in app.  Regarding the capital letters, I will change it